Pipfile

```diff
@@ -4,15 +4,14 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pylint = "*"
-pytest = "*"
-pytest-mock = "*"
-pre-commit = "*"
-
-[packages]
-aws-sam-cli = "*"
-boto3 = "*"
-botocore = "*"
+awscli = "~=1.18.136"
+aws-sam-cli = "~=1.2.0"
+importlib-metadata = "~=1.7.0"
```

Comment:
why is `importlib-*` needed?

Reply:
Hmmm I hadn't read the error messages carefully and I assumed that something in our code was relying on it. It actually looks like pre-commit is [relying on importlib-resources](https://travis-ci.com/github/Sage-Bionetworks-IT/cfn-explode-macro/builds/184261785), but it doesn't get installed as a dependency for some reason

Reply:
Also, SAM CLI seems to [depend on importlib-metadata](https://travis-ci.com/github/Sage-Bionetworks-IT/cfn-explode-macro/builds/184251402#L243).

Reply:
Looks like others have faced a similar problem: https://github.com/pre-commit/pre-commit/issues/853

How about I try removing these dependencies and updating the Python version to 3.8 ?

Reply:
I had the same problem with pre-comit and solved it by bumping python from 3.6 to 3.7 in .travis.yml (in the lambda-budgets repo)

Reply:
Didn't work

Reply:
@zaro0508 could we just leave the importlib-* imports in? It seems to be the least kludge-y way of fixing this problem.